react/features/toolbox/components/web/Toolbox.js

```diff
@@ -1008,6 +1036,13 @@ class Toolbox extends Component<Props, State> {
                     key = 'stats'
                     onClick = { this._onToolbarOpenSpeakerStats }
                     text = { t('toolbar.speakerStats') } />,
+            this._shouldShowButton('embedmeeting')
+                && <OverflowMenuItem
```

Comment:
Is this something we really want? We don't have 2 ways to trigger the password setting procedure, thus I'd expect this button not to exist and just be part of the invite dialog. Can you please double check?

Reply:
As per Tibi and the provided ux, there. should be an entry in the menu as well